### ISSUE TITLE
[Adaptive Cards-ShowCardWizard.JSON]: "Please provide the following information" text is not associated with the first expandable button.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9245]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0
Screen reader: Narrator/NVDA

**Repro steps:**

1. Turn on Narrator through Ctrl + Windows + Enter key.
2. Open the Adaptive Card Visualizer application.
3. Navigate to ShowCardWizard.json button by using Tab key and activate it.
4. ShowCardWizard.json adaptive card will be appeared on right sight of the pane.
5. Navigate to the '1. Name' button using tab key.
6. Observe the issue.

**Actual Result:**
"Please provide the following information" text is not associated with the first expandable i.e., '1.Name' button.

**Expected Result:**
"Please provide the following information" text should be associated with the first expandable i.e., '1.Name' button.

**User Impact:**
Assistive technology users will find it difficult to interact with the UI if no label info is provided.

**WCAG Reference link:**
https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/77f0b27b-b2a0-47a8-9fe3-8562ffe9179a

![Image](https://github.com/user-attachments/assets/89bc2c66-e9a8-4df1-afce-39e0794cf5fa)

### Card JSON

```JSON
N/A
```

**Comment by vagpt** [comment]
Since the product is getting deprecated, hence closing this issue.
